src/test/routes/single-justice-procedure.test.ts

```diff
@@ -6,7 +6,7 @@ describe('Single Justice Procedure', () => {
   describe('on GET', () => {
     test('should return single-justice-procedure page', async () => {
       await request(app)
-        .get('/single-justice-procedure')
+        .get('/summary-of-publications?courtId=0')
```

Comment:
is this a single justice procedure?? this is summary of publications pertaining to a SJP, not the SJP page right? this is just the same SOP page but for a specific court